src/server.c

```diff
@@ -6417,7 +6417,8 @@ void dismissMemory(void* ptr, size_t size_hint) {
 
 /* Dismiss big chunks of memory inside a client structure, see dismissMemory() */
 void dismissClientMemory(client *c) {
-    /* Dismiss client query buffer. */
+    /* Dismiss client query buffer and static reply buffer. */
+    dismissSds(c->buf);
```

Comment:
crash CI in 32bit: https://github.com/redis/redis/runs/5473064515?check_suite_focus=true
`c->buf` is not `sds`.